packages/server/test/e2e/8_read_only_fs.ts

```diff
@@ -0,0 +1,55 @@
+import * as fs from 'fs'
+import * as path from 'path'
+
+const e2e = require('../support/helpers/e2e')
+const chmodr = require('chmodr')
+
+describe('e2e project dir access', function () {
```

Comment:
this is a nice, robust test. you should rename the file to `8_read_only_fs_spec.ts` to match how other e2e tests are named.